pgportfolio/learn/network.py

```diff
@@ -97,7 +97,7 @@ def _build_network(self, layers):
                 network = network[:, :, 0, 0]
                 btc_bias = tf.zeros((self.input_num, 1))
                 network = tf.concat([btc_bias, network], 1)
-                self.voting = network
+                # self.voting = network # is it garbage?
```

Comment:
Actually it's not redundant, the voting result sometimes being printed out.

Reply:
Yes, this is useful in our internal version and could also be used here for debugging(or other purposes in future).

 @lytkarinskiy Thanks for your contribution.
I wonder if you can cancel this single commit so that I can merge the pull request.